build/stf-run-ci/tasks/create_builds.yml

```diff
@@ -19,33 +19,50 @@
 - when: build_config_lookup.resources | length == 0
   block:
   - name: Create BuildConfig and ImageStream
-    shell: oc new-build -n "{{ namespace }}" --name {{ artifact.name }} --dockerfile - < {{ artifact.working_build_dir }}/{{ artifact.dockerfile_path }}
+    ansible.builtin.shell: oc new-build -n "{{ namespace }}" --name {{ artifact.name }} --dockerfile - < {{ artifact.working_build_dir }}/{{ artifact.dockerfile_path }}
 
   - name: Kill first build since it will always fail (triggered on BuildConfig creation)
-    shell: sleep 10 ; oc delete build {{ artifact.name }}-1 -n "{{ namespace }}"
+    ansible.builtin.shell: sleep 10 ; oc delete build {{ artifact.name }}-1 -n "{{ namespace }}"
```

Comment:
Does this still "always fail"? Do we know why?

Reply:
Unknown.
I want to get the job running first, and then we can test efficiencies.

The current job is only deploying from bundles, since I spend a long time trying to get the job working without knowning that the environment was the issue. I'm going to work backwards from running from upstream bundles to running locally-generated bundles to  building the images, I think.

Reply:
Thanks, that explanation makes sense, I think. It doesn't match what I'd expect from reading the docs, but it does match what I've experienced playing with this. Now that you've explained it perhaps I can notice something next time I'm looking at it live.